dev/VSIX/Extension/ProjectReunion.Extension.csproj

```diff
@@ -48,6 +48,9 @@
     <Content Include="LICENSE">
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <Content Condition="'$(EnableExperimentalVSIXFeatures)'=='true'" Include="ExtensionPack.vsext">
```

Comment:
what a great way to implement this and other future optional bits (might carry this over to WTS)